### ISSUE TITLE
Simple txt2img workflow fails to parse

### ISSUE THREAD

**Issue by oxysoft** [rvion/CushyStudio#70]
[hamburger.json](https://github.com/rvion/CushyStudio/files/13322509/hamburger.json)

Please see the attached hamburger workflow.

**Comment by rvion** [comment]
want to join today's stream on https://discord.gg/GfAN6hF2ad ?
I'm going to fix this one live right now

**Comment by rvion** [comment]
Can't seem to find your `lcm-optimizer-sd15.safetensors` on the net.
any link to share (didn't found the file from https://latent-consistency-models.github.io/ )

without it, not quite sure I'm debugging the right thing because of this error
![image](https://github.com/rvion/CushyStudio/assets/2150990/fea355d1-47a9-4381-8324-b0ea2a74496b)

**Comment by oxysoft** [comment]
These weights https://huggingface.co/latent-consistency/lcm-lora-sdv1-5/blob/main/pytorch_lora_weights.safetensors I think you can just put in any LoRA tho, the output won't look right but that's fine for this repro

**Comment by oxysoft** [comment]
Yeah so actually I think literally every single image I ever rendered in ComfyUI doesn't open in CushyStudio with this same bug, so this is critical to figure out. Here is a simplified burger to help you

[simplified-burger.json](https://github.com/rvion/CushyStudio/files/13326393/simplified-burger.json)

**Comment by rvion** [comment]
Thanks ! I'll check this one out tomorrow

**Comment by rvion** [comment]
![image](https://github.com/rvion/CushyStudio/assets/2150990/b2542b1f-00b9-489a-864a-f83c0467e235)

I think I'm onto something 🤣

**Comment by rvion** [comment]
I guess I just have to prioritize adding validation everywhere so I don't send time debugging things anymore, and just wait for people to send me nice screenshots of what error message they see 🙃

**Comment by oxysoft** [comment]
Haha yes indeed, game development taught me you can never have too much printing. I didn't like that error message very much myself, you should know exactly which node crashed it, or if it was outside of that, etc.

**Comment by rvion** [comment]
@oxysoft 

honestly, I have no idea if I have fixed your issue or not, your burger.json never ran on my end.
Your "random prompt" node simply don't work on my ComfyUI install, with everything up-to-date.

Can you update your nodes and send me some other example with both Comfy and your node pack up to date ?

![image](https://github.com/rvion/CushyStudio/assets/2150990/ff2a4eea-fc16-49fc-83bf-ff8dbd00c36e)

when I replace this node by a text primitive, it works.

![image](https://github.com/rvion/CushyStudio/assets/2150990/dda1a038-d8d3-4655-a222-30d829556a87)
